tests/test_module.py

```diff
@@ -32,4 +32,24 @@ def test_passes_if_module_has_ref_github(self, caplog):
     def test_passes_if_module_not_git(self, caplog):
         file = 'tests/test_module/module_notgit.tf'
         with Wrap(self, [file], expect_exit=False):
-            assert ('[FAIL] github_module_ref:Modules sourced from GitHub should be pinned:some_module:{}'.format(file)) not in caplog.text
+            assert ('Modules sourced from GitHub should be pinned FAIL in {}'.format(file)) not in caplog.text
```

Comment:
Its not letting me add the comment on the latest commit, but looks like the master merge cleared this out.  Should update back to something like:

```suggestion
            assert ('[PASS] github_module_ref:Modules sourced from GitHub should be pinned:some_module:{}'.format(file)) in caplog.text
```

Reply:
I knew I shouldn't have done a merge, and should have stuck with my trusty rebase :)